pages/clubs-curriculum.jsx

```diff
@@ -0,0 +1,186 @@
+var React = require('react');
+var ImageTag = require('../components/imagetag.jsx');
+var Router = require('react-router');
+var Link = Router.Link;
+
+var CurriculumIntro = React.createClass({
+  render: function () {
+    return (
+        <section className="row intro">
+          <h1 className="col-sm-12">Clubs Curriculum</h1>
+          <div className="col-sm-3 col-md-3 col-md-offset-1 col-lg-3">
+            <ImageTag alt="Woman training a young man on a computer"
+                className="img-circle img-scale-to-fit" src1x="/img/clubs-curriculum-page/photo-clubs-curriculum.jpg"
+                src2x="/img/clubs-curriculum-page/photo-clubs-curriculum@2x.jpg"/>
+          </div>
+          <h2 className="col-sm-9 col-md-8">
+            Activities to teach the web in your club. Developed and tested by our seasoned educator community. Easy
+            to use, guaranteed fun.
+          </h2>
+        </section>
+    );
+  }
+});
+
+var WebLitBasics = React.createClass({
+  render: function () {
+    return (
+        <section className="row web-lit-basics">
+          <div className="col-sm-12">
+            <h2>Web Literacy Basics</h2>
+            <p>
+              Learners get familiar with reading, writing and participating on the web in this six-part module.
+              Discover the foundations of the web through production and collaboration. The learning objectives
+              underpinning each activity are informed by
+              Mozilla&rsquo;s <a href="https://webmaker.org/en-US/resources">Web Literacy Map</a>
```

Comment:
I would double check with someone else that we don't want this linked to learn-like-mozilla/web-literacy instead